Ports/RetroArch/package.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=RetroArch
+useconfigure="true"
+version="1.10.3"
+archive_hash="2af44294e55f5636262284d650cb5fff55c9070ac3a700d4fa55c1f152dcb3f2"
+files="https://github.com/libretro/${port}/archive/refs/tags/v${version}.tar.gz ${port}-${version}.tar.gz $archive_hash"
+auth_type=sha256
+depends=("freetype" "SDL2" "zlib")
+
+configopts=(
+    "--disable-builtinglslang"
+    "--disable-discord"
+    "--disable-glsl"
+    "--disable-glslang"
+    "--disable-opengl"
+    "--disable-slang"
+    "--disable-spirv_cross"
+    "--disable-systemmbedtls"
+    "--disable-update_cores"
+)
+
+launcher_name=RetroArch
+launcher_category=Games
+launcher_command=/usr/local/bin/retroarch
+icon_file=media/retroarch.ico
+
+function pre_configure() {
+    export CFLAGS="-I${SERENITY_INSTALL_ROOT}/usr/include/LibGL -I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2"
```

Comment:
Does RetroArch not find SDL2 automatically?

Reply:
If I don't add the include I got this error compilation

Reply:
If I don't add the include I get this error as it tries with the one from the system

```
In file included from gfx/drivers/sdl2_gfx.c:37:
/usr/include/SDL2/SDL_syswm.h:69:10: fatal error: X11/Xlib.h: No such file or directory
   69 | #include <X11/Xlib.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:206 : obj-unix/release/gfx/drivers/sdl2_gfx.o] Erreur 1
make: *** Attente des tâches non terminées....
In file included from gfx/common/sdl2_common.c:29:
/usr/include/SDL2/SDL_syswm.h:69:10: fatal error: X11/Xlib.h: No such file or directory
   69 | #include <X11/Xlib.h>
      |          ^~~~~~~~~~~~
```